src/read/pe/export.rs

```diff
@@ -0,0 +1,149 @@
+use alloc::vec::Vec;
+use core::convert::TryInto;
+use core::fmt::Debug;
+
+use crate::read::ReadError;
+use crate::read::Result;
+use crate::ByteString;
+use crate::{pe, Bytes, LittleEndian as LE, U16Bytes, U32Bytes};
+
+/// Where an export is pointing to
+#[derive(Clone)]
+pub enum Target<'data> {
```

Comment:
```suggestion
pub enum ExportTarget<'data> {
```
Because otherwise this appears as `read::pe::Target` which seems too general.